src/map/modifier.h

```diff
@@ -620,6 +620,9 @@ enum MODIFIER
     // For example, +10% damage to Chant du Cygne would be ID 570 + 225 (795)
     MOD_WEAPONSKILL_DAMAGE_BASE   = 570,
 
+	// Damage/reistance bonus from Circle Effects
+	MOD_ANCIENT_CICRLE_BONUS	  = 571, // Increases/decreases damage against/from dragons
```

Comment:
use the first mod_spare (64), and remove the commented out mod_spare - 571 is already in use for combo dmg%